### ISSUE TITLE
Diamond help page content needs updating, including alt txt for images

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/scigateway#815]
**Description**:
Automated Accessibility checker found that the Images on the contact page don't have aria labels. 

![Screenshot 2021-11-12 131218](https://user-images.githubusercontent.com/83226114/141472648-3499eb22-1c08-4c1c-930c-32340749466a.png)
**Acceptance criteria**:

- [ ] A checklist
- [ ] of criteria to accept this story as done

**Comment by sam-glendenning** [comment]
Note that this is currently a Diamond-specific issue - ISIS is unaffected. Help page content exists in a language file specifically for each configuration. ISIS' language file, and therefore help page, is up to date. Diamond currently has the old TopCAT help page and images (including no alt text). This is what this issue should concern and must tackle

**Comment by louise-davies** [comment]
This has been completed
